aspnetcore/toc.yml

```diff
@@ -351,6 +351,8 @@ items:
                     uid: blazor/hybrid/tutorials/wpf
               - name: Routing and navigation
                 uid: blazor/hybrid/routing
+              - name: Developer tools
+                uid: blazor/hybrid/developer-tools
```

Comment:
```suggestion
              - name: Accessing Browser Developer tools
                uid: blazor/hybrid/accessing-browser-developer-tools
```

Reply:
Do we want to make it this specific? I mean this scopes down to just "accessing" and not "using" or doing anything else. I generally try to keep these generic and broad so that we can place any kind of "developer tools" guidance.

Another possibility is to make the whole topic **_even more generic_** by making this guidance part of a *Troubleshoot*-type of topic or an *Error handling*-type of topic.

Reply:
I like `error handling`. I'm a bit hesitant to do dev tools as users may think this is guidance for vs/vscode which it is not (yet?). There are a few other things which may be upcoming on the error handling front so this would be a good section for that as well.

Reply:
Ok ... 👂 good to me. I'll pitch a revision for it on Thursday morning with the additional updates. I'll ping u back tomorrow.